layout/units.md

```diff
@@ -14,28 +14,56 @@ This lesson is an introduction and reference guide to some units of measurement
+
+```html
+<html>  <!-- the root element -->
+  <body>
+
+    <style>
+      html {
+        font-size: 16px; <!-- default font size for most web browsers -->
+      }
+    </style>
+
+    <header> <!-- parent element -->
+      <h1>Hello world!</h1>
+    </header>
 
-Percentages can be used in measurements that will be a percentage relative to another value. This is done by adding `%` to the end of the value.
+  </body>
+</html>
+```
 
-What `100%` means depends on the given HTML and context.  For example the width and height are used to scale content relative to an element's container.
+### Percentages
 
-#### `em` or `rem`
+Percentages can be used in measurements that will be a percentage relative to another value. This is done by adding `%` to the end of the value of a CSS property.
+
+What `110%` means depends on the given HTML and context.  If our `h1` tag above had the styling `font-size: 110%`, the font size would be 110% larger than its `parent element`.
+
+### `em` or `rem`
 
 `em` and `rem` are relative measurements based on the height of a font.
 
-One `em` unit (`1em`) is equivalent to the size of the font in the element.
+One `em` unit (`1em`) is equivalent to the size of the font in the parent element. If our `h1` tag above had the styling `font-size: 1.2em`, the font would be **1.2 times** the size of its `parent element`.
 
-One `rem` unit (`1em`) is equivalent to the size of the font defined in the root element of the DOM, meaning the `html` element.
+One `rem` unit (`1rem`) is equivalent to the size of the font defined in the root element of the DOM, meaning the `html` element. If our `h1` tag above had the styling `font-size: 2rem`, the font would be **twice** the size of its `root element`.
 
-#### `vw` and `vh`
+### `vw` and `vh`
 
 The _viewport_ is the area of screen that is displaying the web page. It's the literal area of a web page visible at a given moment, which excludes overflowing sections of the HTML that are "off-screen."
 
 This is distinct but similar to the device screen or browser window, as both affect the viewport.
 
-One `vw` (`1vw`) is equal to 1% of the viewport width, and `100vw` is 100% of the viewport width.
+One `vw` (`1vw`) is equal to 1% of the viewport width, and `100vw` is 100% of the viewport width. If our `h1` tag above had the styling `width: 25vw`, the _element_ would take up 25% of the viewport's width.
+
+`vh` is relative to the viewport's height. One `1vh` (`1vh`) is equsl to 1% of the viewport height, and `100vh` is 100% of the viewport height. If our `h1` tag above had the styling `font-size: 100vh`, the _font size_ would take up the entire height of the viewport.
+
+### Live Preview
 
-`vh` is relative to the viewport's height. `1vh` is 1% of the viewport height, and `100vh` is 100%.
+Here is an [interactive example](https://codepen.io/adadev/pen/yLovVXz) using _Codepen_ to play around with! Try out different units and values to see what happens!
```

Comment:
I suggest naming the text in the elements to match their class names, to make it easier to spot what's happening.

Reply:
done and done!